mobile/src/common/types/navigation/navigation.ts

```diff
@@ -1,3 +1,4 @@
 export { type NavigationScreenProps } from './navigation-screen-props.type';
 export { type RootNavigationParamList } from './root-navigation-param-list.type';
 export { type RootNavigationScreenProps } from './root-navigation-screen-props.type';
+export { type ParamListBase } from '@react-navigation/native';
```

Comment:
We don't need this type in common types and we have redeclared it, it equals RootNavigationParamList